src/site/content/en/blog/schemeful-samesite/index.md

```diff
@@ -0,0 +1,377 @@
+   <td>
+   </td>
+   <td><strong>HTTP → HTTPS</strong>
+   </td>
+   <td><strong>HTTPS → HTTP</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><code>SameSite=Strict</code>
+   </td>
+   <td>⛔ Blocked
+   </td>
+   <td>⛔ Blocked
+   </td>
+  </tr>
+  <tr>
+   <td><code>SameSite=Lax</code>
+   </td>
+   <td>⛔ Blocked
+   </td>
+   <td>⛔ Blocked
+   </td>
+  </tr>
+  <tr>
+   <td><code>SameSite=None;Secure</code>
+   </td>
+   <td>✓ Allowed
+   </td>
+   <td>⛔ Blocked
+   </td>
+  </tr>
+</table>
+
+## How can I test my site?
+
+Developer tooling and messaging are available in Chrome and Firefox.
+
+From Chrome 86, the [Issue tab in
+DevTools](https://developers.google.com/web/tools/chrome-devtools/issues) will
+include Schemeful Same-Site issues. You may see the following issues highlighted
+for your site.
+
+Navigation issues:
+  - "Migrate entirely to HTTPS to continue having cookies sent on same-site
+    requests"—A warning that the cookie **will be** blocked in a future version
+    of Chrome.
+  - "Migrate entirely to HTTPS to have cookies sent on same-site requests"—A
+    warning that the cookie **has been** blocked.
+
+Subresource loading issues:
+  - "Migrate entirely to HTTPS to continue having cookies sent to same-site
+    subresources" or "Migrate entirely to HTTPS to continue allowing cookies to
+    be set by same-site subresources"—Warnings that the cookie **will be**
+    blocked in a future version of Chrome.
+  - "Migrate entirely to HTTPS to have cookies sent to same-site subresources"
+    or "Migrate entirely to HTTPS to allow cookies to be set by same-site
+    subresources"—Warnings that the cookie **has been** blocked. The latter
+    warning can also appear when POSTing a form.
+
+More detail is available in [Testing and Debugging Tips for Schemeful
+Same-Site](https://www.chromium.org/updates/schemeful-same-site/testing-and-debugging-tips-for-schemeful-same-site).
+
+From Firefox 79, with `network.cookie.sameSite.schemeful` set to `true` via
+`about:config` the console will display message for Schemeful Same-Site issues.
+You may see the following on your site:
+
+- "Cookie `cookie_name` **will be soon** treated as cross-site cookie against
+  `http://site.example/` because the scheme does not match."
+- "Cookie `cookie_name` **has been** treated as cross-site against
+  `http://site.example/` because the scheme does not match."
+
+## FAQ
+
+### My site is already fully available on HTTPS, why am I seeing issues in my browser's DevTools?
+
+It's possible that some of your links and subresources still point to insecure
+URLs.
+
+One way to fix this issue is to use [HTTP
+Strict-Transport-Security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security)
+(HSTS) and the `includeSubDomain` directive. With HSTS + `includeSubDomain` even
+if one of your pages accidentally includes an insecure link the browser will
+automatically use the secure version instead.
+
+### What if I can't upgrade to HTTPS?
```

Comment:
@rowan-m & @mihajlija I have a non-blocking requested change related to this section. See https://github.com/GoogleChrome/web.dev/issues/4278

Reply:
Ack, good call. I can do some restructuring to tackle this and the Alex feedback.